src/libs/ReportUtils.js

```diff
@@ -2138,10 +2138,13 @@ function getChatByParticipantsByLoginList(participantsLoginList) {
     participantsLoginList.sort();
     return _.find(allReports, (report) => {
         // If the report has been deleted, or there are no participants (like an empty #admins room) then skip it
-        if (!report || !report.participants || isThread(report)) {
+        if (!report || (report.participantAccountIDs || []).length === 0 || isThread(report)) {
             return false;
         }
 
+        // TODO: update to use report.participantAccountIDs
+        // Note: To migrate we have to make sure the "duplicate chat report" issue doesn't come back:
+        // https://github.com/Expensify/App/issues/21184
```

Comment:
MAIN TODO WE'RE BLOCKED ON IS HERE

Reply:
Nevermind - this case will be handled here: https://github.com/Expensify/App/issues/21184

- so it's expected we still access `report.participants` here